lib/commands/deploynodeapp.js

```diff
@@ -72,6 +80,31 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
+    name: 'Runtime to deploy the Node.js app to',
+    shortOption: 'r',
+    required: false
+  },
+  'node-version': {
```

Comment:
This should be `runtime-version`, not `node-version`.  We don't want to have to change this when we get multi-language support.